connectors/opa/Dockerfile

```diff
@@ -1,7 +1,18 @@
-FROM gcr.io/distroless/static:nonroot
+#FROM golang:1.13.8
+FROM golang:1.13.8-alpine
 
-COPY opa-connector /
+WORKDIR /m4d/
+COPY go.mod go.mod
+
+COPY pkg/connectors/protobuf pkg/connectors/protobuf
+
+WORKDIR /m4d/connectors/opa
+COPY connectors/opa .
+
+RUN go get -v ./...
+
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -a -o opa-connector *.go
 
 EXPOSE 50082
 
-ENTRYPOINT ["/opa-connector"]
+CMD ["./opa-connector"]
```

Comment:
Explain why this was changed please. The rest of the project tries to build go stuff out of docker (improved CI performance and minimal rootless images).

Reply:
Restored Dockerfile to the version in master branch. Thanks.